docs/conf/portland/2022/example-proposal.rst

```diff
@@ -0,0 +1,60 @@
+:template: {{year}}/generic.html
+
+Example talk proposal title
```

Comment:
Wondering if we should ask folks to explicitly state content warnings in their proposals? i.e pandemic etc...